README.md

```diff
@@ -1,17 +1,48 @@
 # Xpense
 
-The Xpense project is an open source platform for housekeeping.
+The **Xpense** project is an open source platform for housekeeping.
 Its main objective is to monitor housekeeping related income and expenses.
 In addition, it provides the option to share data anonymously, thereby
 providing others the opportunity to compare and learn from families in the same status group.
 
-# Team members
+## Addressing the Challenge
+
+Families and households are going into financial distress and
+the stigma of talking about the finance in the house is
+considered taboo.
+
+**Xpense** provides the option to monitor and share data
+anonymously, thereby providing others the opportunity to compare
+and learn from people facing the same financial challenges.
+
+We think that by monitoring and learning from examples of more
+experienced people, the financial household management crisis of
```

Comment:
You can't assume someone is more experienced, as a user you'l look at anonymous data, one can't tell if this data is coming from more or less experienced people.
Also, the data can hold information which is not better than what the user is having.
Try to rephrase this line.

Reply:
I fixed it